docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md

```diff
@@ -294,13 +306,13 @@ module "manually_uploaded_housing_data_to_raw" {
 
 [aws_cron_expressions]: https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html#CronExpressions
 [prerequisites]: ../../ingesting-data/google-sheets-import#prerequisites
-[terraform-directory]: https://github.com/LBHackney-IT/Data-Platform/tree/ac2a8d56e77ba94f1123fcdb983387081887a72f/terraform
+[terraform-directory]: https://github.com/LBHackney-IT/Data-Platform/tree/main/terraform
 [jobs-directory]: https://github.com/LBHackney-IT/Data-Platform/tree/main/scripts/jobs
-[adding-script]: #1-add-your-script-to-the-data-platform-project
+[adding-script]: #1-add-your-script-to-the-data-platform-project-using-the-github-ui
 [using-glue-studio]: ./using-glue-studio
 [cron-expression-generator]: https://www.freeformatter.com/cron-expression-generator-quartz.html
 [glue-worker-types]: https://docs.aws.amazon.com/glue/latest/dg/add-job.html#:~:text=Own%20Custom%20Scripts.-,Worker%20type,-The%20following%20worker
 [helpers-folder-github]: https://github.com/LBHackney-IT/Data-Platform/tree/main/scripts/helpers
 [committing-changes]: ../../getting-set-up/using-github#committing-your-changes-to-the-data-platform-project
```

Comment:
Is this still wrong?

Reply:
No, it's the correct url
![image](https://user-images.githubusercontent.com/70905620/144022222-c69567bb-d967-409f-891f-928244208445.png)